### ISSUE TITLE
Multiple examples highlighted as syntax error

### ISSUE THREAD

**Issue by andyg101** [rlogiacco/Natural#45]
I believe that the following is valid gherkin syntax. However the second examples block is highlighted as a syntax error.

``` Cucumber
Feature: Test feature
Scenario Outline: Test scenario 1
Given i have <something>
When I don't need <anything>
Then complain   

  Examples: Something
    |something|anything|
    |foo      |bar     |

  Examples: Nothing
    |something|anything|
    |foo      |bar     |
```
##

**Comment by James-Y-Du** [comment]
I met the same problem. Is someone looking into this issue right now?

**Comment by rlogiacco** [comment]
No, but you are welcome to contribute a fix for it.

**Comment by drkstr101** [comment]
Fixed in https://github.com/rlogiacco/Natural/pull/73

**Comment by rlogiacco** [comment]
#73 was closed, did we incorporate this into #82? I'm getting confused by all these branches/PRs 😄

**Comment by drkstr101** [comment]
Yes indeed!  This issue can be closed as done.

**Comment by rlogiacco** [comment]
fixed in #82
